<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LTIN508596" failures="3" tests="3" name="chrometest" time="65.764" errors="0" timestamp="2024-04-18T22:14:36 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="13.796">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: stale element not found
  (Session info: chrome=120.0.6099.130)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: &amp;apos;4.18.1&amp;apos;, revision: &amp;apos;b1d3319b48&amp;apos;
System info: os.name: &amp;apos;Windows 11&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.10&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [cb8d01222675abbc17901ead8bfeea07, clickElement {id=79D83C81D30069941E7ECA2C6B7E9944_element_123}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52264}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52264/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (cb8d01222675abbc17901ead8bfeea07)] -&amp;gt; xpath: //input[@id=&amp;apos;country&amp;apos;]]
Session ID: cb8d01222675abbc17901ead8bfeea07">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: stale element not found
  (Session info: chrome=120.0.6099.130)
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Build info: version: '4.18.1', revision: 'b1d3319b48'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.10'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [cb8d01222675abbc17901ead8bfeea07, clickElement {id=79D83C81D30069941E7ECA2C6B7E9944_element_123}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 120.0.6099.130, chrome: {chromedriverVersion: 120.0.6099.109 (3419140ab66..., userDataDir: C:\Windows\SystemTemp\scope...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52264}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52264/devtoo..., se:cdpVersion: 120.0.6099.130, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on windows (cb8d01222675abbc17901ead8bfeea07)] -> xpath: //input[@id='country']]
Session ID: cb8d01222675abbc17901ead8bfeea07
at org.openqa.selenium.remote.ErrorCodec.decode(ErrorCodec.java:167)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:138)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:519)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:223)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:76)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy28.click(Unknown Source)
at pageObjects.TravelInsurance.clickCountry(TravelInsurance.java:107)
at StepDefinitions.TC_001_TravelInsurance.user_enters_destination_details(TC_001_TravelInsurance.java:26)
at ✽.User enters destination details(file:///C:/Users/2327192/Downloads/Hackathon_TravelInsurance/Hackathon_TravelInsurance/Features/Feature1_TravelInsurance.feature:6)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="36.825">
    <failure type="java.io.FileNotFoundException" message="C:\Users\2327192\Downloads\Hackathon_TravelInsurance\Hackathon_TravelInsurance\testdata\ExcelWrite.xlsx (The process cannot access the file because it is being used by another process)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\2327192\Downloads\Hackathon_TravelInsurance\Hackathon_TravelInsurance\testdata\ExcelWrite.xlsx (The process cannot access the file because it is being used by another process)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
at utilities.ExcelUtility.setcelldata(ExcelUtility.java:39)
at pageObjects.CarInsurance.printErrorMessage(CarInsurance.java:127)
at StepDefinitions.TC_002_CarInsurance.user_fills_incorrect_user_details_and_gets_error_message(TC_002_CarInsurance.java:43)
at ✽.User fills incorrect user details and gets error message(file:///C:/Users/2327192/Downloads/Hackathon_TravelInsurance/Hackathon_TravelInsurance/Features/Feature2_CarInsurance.feature:8)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.51">
    <failure type="java.io.FileNotFoundException" message="C:\Users\2327192\Downloads\Hackathon_TravelInsurance\Hackathon_TravelInsurance\testdata\ExcelWrite.xlsx (The process cannot access the file because it is being used by another process)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\2327192\Downloads\Hackathon_TravelInsurance\Hackathon_TravelInsurance\testdata\ExcelWrite.xlsx (The process cannot access the file because it is being used by another process)
at java.base/java.io.FileOutputStream.open0(Native Method)
at java.base/java.io.FileOutputStream.open(FileOutputStream.java:293)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:235)
at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
at utilities.ExcelUtility.setcelldata(ExcelUtility.java:39)
at pageObjects.HealthInsurance.printHealthInsurancePlans(HealthInsurance.java:31)
at StepDefinitions.TC_003_HealthInsurance.user_displays_all_health_insurance_plans(TC_003_HealthInsurance.java:25)
at ✽.User displays all health insurance plans(file:///C:/Users/2327192/Downloads/Hackathon_TravelInsurance/Hackathon_TravelInsurance/Features/Feature3_HealthInsurance.feature:6)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- chrometest -->
